TestCases/cont_adj_euler/wedge/inv_wedge_ROE_multiobj.cfg

```diff
@@ -362,7 +377,9 @@ VISUALIZE_DEFORMATION= YES
 %
 % Optimization objective function with scaling factor
 % ex= Objective * Scale
-OPT_OBJECTIVE=LIFT*-1.0E4;OUTFLOW_GENERALIZED*0.01 
+OPT_OBJECTIVE=AVG_TOTAL_PRESSURE*1E-8;(DRAG = 0.045)*1E4
```

Comment:
This is an example of setting the optimization objective function to include a penalty function. 
In general, I was trying to set it up so that the user can specify the constraint in the same way that they would if the constraint was passed explicitly to the optimizer. 

Using (Drag = constraint)*scale means that rather than optimizing to minimize drag, the optimization function will include scale*(CD - constraint)^2. Using an inequality would do something similar, except that when the inequality constraint is satisfied the penalty function is 0.

See obj_p and obj_dp in eval/design.py for where the penalty function is implemented. The chain rule is used to obtain the gradients needed by the optimizer.